yt/utilities/io_handler.py

```diff
@@ -12,6 +13,26 @@
 
 use_caching = 0
 
+FluidFieldType = str
+FluidFieldName = str
+FluidFieldTuple = typing.Tuple[FluidFieldType, FluidFieldName]
```

Comment:
This feels pretty more complicated than necessary
```suggestion
FluidFieldTuple = Tuple[str, str]
```